packages/synapse-react-client/src/components/entity/metadata/AnnotationsTable.integration.test.tsx

```diff
@@ -75,4 +75,6 @@ describe('AnnotationsTable tests', () => {
     renderComponent()
     await screen.findByText('This Project has no annotations.')
   })
+
+  it.todo('Handles scenario where derived annotations are pending')
```

Comment:
Held off on this fairly complex test in case this component is going to be redesigned